transform/voxel_around_line_test.go

```diff
@@ -13,6 +13,44 @@ import (
 	"github.com/trajectoryjp/spatial_id_go/v2/shape"
 )
 
+func TestGetSpatialIdsWithinRadiusOfLine(t *testing.T) {
```

Comment:
This test will pass without merging this pull request.
Modify this test and prove that the issue is resolved.

Reply:
when GetSpatialIdsWithinRadiusOfLine(..., skipsMeasurement=true) the test fails at commit 46c3a4103867fbf9099f42f2d70d18f401b40126

Reply:
@buscandoaverroes
I checked it out.
Thank you!